src/glwidget.cpp

```diff
@@ -639,6 +639,7 @@ int GLWidget::reconfigure(bool isMulti)
             m_consumer->set("0.prefill", 1);
             if (property("keyer").isValid())
                 m_consumer->set("0.keyer", property("keyer").toInt());
+            m_consumer->set("1.channels", property("audio_channels").toInt());
```

Comment:
The multi consumer is used when using decklink for monitoring. Unfortunately, the DeckLink API only accepts the values 2, 8, or 16 which does not match our multi-channel use cases well at all. I found out because 6 was failing to initialize and looked it up. I was able to hack it to use 8, but the audio was very distorted, and my receiver could not understand it (even though it is 7.1).

Setting "1.channels" here to target decklink does not work because it is overridden by the multi consumer (all the consumers need to be in sync on this parameter). Really, for now, we need to hide the Audio Channels in the Settings menu when using a decklink for external monitoring.

Reply:
Oh, yes. That is true. I think a more comprehensive solution (in the future) would be to extend the Decklink consumer to configure the device for as many channels as necessary to fit the requested channels - and then fill the rest of the channels with silence. This is a method I have used in the past to get around the Decklink limitations.

Your suggestion is more appropriate in the short-term. I'll look into it.